conf/nginx.conf

```diff
@@ -76,8 +76,8 @@ location ^~ __PATH__/ {
     deny all;
   }
 
-  location ~ ^__PATH__/(?:index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|oc[ms]-provider\/.+|.+\/richdocumentscode\/proxy)\.php(?:$|\/) {
-    fastcgi_split_path_info ^(.+?\.php)(\/.*|)$;
+  location ~ ^__PATH__/(?:index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|oc[ms]-provider\/.+|.+\/richdocumentscode\/proxy)\.php(\/.*|)$ {
```

Comment:
Could you explain why you need this change ? I remeber there was something around this on "15 to 18" PR

Reply:
From what I recall, otherwise accessing through the web browser just wasn't working... I couldn't figure out why (as it's the recommended configuration), so I got back to what worked...